tools/EntityFramework.settings.targets

```diff
@@ -24,7 +24,8 @@ Project global pre-settings.
     <VersionReleasePrefix>0</VersionReleasePrefix>
     <VersionRelease>$(VersionReleasePrefix)-beta2</VersionRelease>
     <VersionReleaseName>Beta 2</VersionReleaseName>
-    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0' OR '$(VisualStudioVersion)' == '14.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '14.0'">4.5.1</ToolingDotNetFrameworkVersion>
```

Comment:
I think I remember reading somewhere that even though it comes with .NET 4.6 most of it needs to run on 4.5 so it can run `devenv /setup` during install without rebooting. ...but that doesn't really apply to the EF6 Tools.